.gitignore

```diff
@@ -6,3 +6,4 @@
 /*.log
 /build-results/
 make-checkout-workdir
+bots
```

Comment:
Why is that necessary? The extra clone of bots should always be in make-checkout-workdir/bots, not in ./bots?

Reply:
The commands outputted by tests-scan is `bots/make-checkout`. But relative to cockpit-project/bots it should be `./make-checkout`. During trying to simulate this I found it easier to just create a bots/ within bots.

But maybe it's worth changing tests-scan so the make-checkout command is relative to tests-scan.

Reply:
Thanks, let's live with the bots/bots thing until bots/ gets dropped from cockpit, and we can clean up the temporary hacks.